nix/internal/unix.nix

```diff
@@ -220,7 +224,24 @@ in
       ln -s ${dolos-configs} $out/dolos-configs
     '';
 
-    testgen-hs-flake = (import inputs.flake-compat {src = inputs.testgen-hs;}).defaultNix;
+    testgen-hs-flake = let
+      unpatched = inputs.testgen-hs;
+    in
+      (import inputs.flake-compat {
+        src =
+          if targetSystem != "aarch64-linux"
+          then unpatched
+          else {
+            outPath = toString (pkgs.runCommand "source" {} ''
+              cp -r ${unpatched} $out
+              chmod -R +w $out
+              cd $out
+              patch -p1 -i ${./testgen-hs--enable-aarch64-linux.diff}
```

Comment:
Wait, but we already have `aarch64-linux` on `main`. Something is off here, another conflict?